lambda_functions/deploy_test_deaenv_js/README.md

```diff
@@ -0,0 +1,67 @@
+## Setup
```

Comment:
I'm not keen on how much duplication there is between `deploy_test_deaenv_js` and `execute_ssh_command_js`.
Especially since the `handler.js` is almost but not quite the same.

I can see that the two directories have lambdas intended at quite different purposes.

Perhaps a nicer way would be to use Serverless's [ability to reference variables in other files](https://serverless.com/framework/docs/providers/aws/guide/variables/#reference-variables-in-other-files) to split up the `serverless.yaml` so that it doesn't get out of hand.

More than happy to leave it like this for now to keep everything rolling along.